site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
```

Comment:
This page is about how to receive mobile push notifications when a corporate proxy server is required - not so much about the corporate proxy server itself.

Would you be open to suggesting a couple of options for a title that best reflects the contents?

Another option may be to include this section within the "Push Notification Service" page https://developers.mattermost.com/contribute/mobile/push-notifications/service/

Reply:
Sure:

- PNS with Corporate Proxy
- Post Proxy Relay
- Corporate Proxy Solutions

Reply:
I like "PNS with Corporate Proxy" @faase 

"Push Notification Service with Corporate Proxy" if it fits in the sidebar,

Reply:
I asked and it should wrap. Using `Push Notification Service with Corporate Proxy`